src/app/rpmostree-override-builtins.cxx

```diff
@@ -90,6 +92,57 @@ handle_override (RPMOSTreeSysroot  *sysroot_proxy,
 
   g_autoptr(GVariant) previous_deployment = rpmostree_os_dup_default_deployment (os_proxy);
 
+  if (override_replace && opt_ex_pin_from_repos)
+    {
+      g_autoptr(GPtrArray) override_replace_final = NULL;
```

Comment:
This needs to be declared outside the if-statement since it needs to be able to outlive that scope (since we use its data in the call to `update_deployment`.